fs.go

```diff
@@ -32,18 +33,7 @@ func IsRegular(name string) (bool, error) {
 }
 
 func IsDir(name string) (bool, error) {
-	// TODO: lstat?
-	fi, err := os.Stat(name)
-	if os.IsNotExist(err) {
-		return false, nil
-	}
-	if err != nil {
-		return false, err
-	}
-	if !fi.IsDir() {
-		return false, errors.Errorf("%q is not a directory", name)
-	}
-	return true, nil
+	return internal.IsDir(name)
```

Comment:
I had to move this to internal because HasFilepathPrefix uses IsDir, and since dep imports internal, I end up with a cycle if I import dep from internal.